hutch_python/plugins/experiment.py

```diff
@@ -11,9 +11,18 @@
 class Plugin(BasePlugin):
     """
     Plugin to load experiment-spefic includes.
+
+    By default, the Experiment plugin attempts to load information from the
+    PCDS Questionnaire. You can disable this by setting, ``questionnaire:
+    False`` underneath the ``experiment`` section in your YAML configuration
     """
     name = 'experiment'
 
+    def pre_plugins(self):
+        if self.info.get('questionnaire', True):
+            from .questionnaire import Plugin as QSPlugin
+            return [QSPlugin(conf=self.conf)]
+
     def get_objects(self):
         expname = self.info['name']
```

Comment:
Can you change this to be looking for `run` and `proposal` and combining them into the `expname`? Otherwise this doesn't actually work with the config we were looking to use for mfx.